pkg/scheduler/plugins/proportion/proportion.go

```diff
@@ -17,11 +17,9 @@ limitations under the License.
 package proportion
 
 import (
-	"math"
-	"reflect"
-
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/klog"
+	"math"
```

Comment:
Please keep the imported package in order.

Reply:
the issue: https://github.com/volcano-sh/volcano/issues/2364

Reply:
The gofmt command was executed before the file was submitted and is now in the correct order